Resources/Help/analyses/ANCOVA.md

```diff
@@ -0,0 +1,227 @@
+- Dependent Variable: The variable you are interested in. Also called the outcome variable. 
+- Fixed Factors: The variables that you manipulated/define the different groups. Also called the independent variables.  
+- Covariates: In this box you can place the variable that is the covariate. Covariates are continuous variables that have an influence on the dependent variable but are not part of the experimental manipulation.  
+- WLS Weights: Weighted Least Squares, here you can place the variable specifying which points have more weight and are therefore considered more informative. For this last option it is important to know the weights a priori. This option is mainly used when the errors are heteroskedastic.    
+
+#### Model: 
+- Components and model terms 
+    - Components: All the independent variables and covariates you can put in the model. 
+    - Model terms: The independent variables and covariates being used in the model. By default all the independent variables and covariates included in the model are placed in this box, plus the interaction between the independent variables.  
+- Sum of Squares 
+    - Type I: Sequential sum of squares. It is the reduction of error when each factor of the model is added to the factors already included, containing the order in the model. 
+    - Type II: Hierarchical/Partially sequential sum of squares. It is the reduction of error when a factor is added to the model that includes all the other factors, except the factors where the added factor is part of. Therefore, it does not include interactions. 
+    - Type III: Partial sum of squares. It is the reduction of error when a factors is added at last to the model, a model that includes all the other factors, including interactions with this factor. This type is selected by default. 
+
+#### Assumption Checks: 
+- Homogeneity tests: By selecting this option it will be checked if the variance of the dependent variable is equal between the groups by using Levene's test.  
+- Q-Q plot of residuals: Checks the validity of the distributional assumption of the data set. More specific, from the plot it could be retrieved if the data is normally distributed and linear. 
+
+#### Contrasts: 
+For every independent variable you can select a specific contrast by clicking on the `none` in the right column.  
+- Factors: These are the independent variables included in the analysis, so the variables placed in the `Fixed Factors` box.  
+- Contrasts: Contrast allow you to conduct planned comparisons, for example to compare the different levels of the independent variable with each other. There are different contrasts that allow for different types of comparisons.  
+    - none: By selecting this option no contrasts are calculated. This option is selected by default. 
+    - deviation: By selecting this contrast the mean of each level of the independent variable is compared with the overall mean (the mean when all the levels are taken together). 
```

Comment:
By selecting this contrast (comma) ... 
This goes for all remaining items, add a comma in the same way